packages/pixelboy/apps/journal/checkin/checkin.coffee

```diff
@@ -47,3 +49,34 @@ class PAA.PixelBoy.Apps.Journal.CheckIn extends AM.Component
 
       else
         FlowRouter.go 'journal'
+
+  getExternalImgUrl: (event) ->
+    url = event.currentTarget.value
+    externalImgUrl = null
+    imgPreview = $('.image-preview')
```

Comment:
All variables that hold jQuery objects start with a `$` for clarity's sake. So this should be `$imagePreview`. `imagePreview` would be the direct underlying DOM element. In sense of `imagePreview = $imagePreview[0]`.